src/hooks/useCredential.js

```diff
@@ -17,7 +17,7 @@ export const login = async (credentials, store, router) => {
   await store
     .dispatch('login', credentials)
     .then(() => {
-      router.push({ name: 'Dashboard' })
+      window.location.href = window.location.href + '/Dashboard'
```

Comment:
not sure what's the reason here but it should be `/dashboard` instead of `/Dashboard` otherwise some tests will probably fail because of that (some tests expect it navigates to dashboard after login)

Reply:
/Dashboard will make it navigate to tradezone instead of dashboard